src/main/java/ip/Duke.java

```diff
@@ -0,0 +1,287 @@
+            switch (taskTypeStr) {
+                case "T":
+                    newTask = new ToDo(description, isDone);
+                    break;
+                case "D":
+                    String by = taskArray[3];
+                    newTask = new Deadline(description, by, isDone);
+                    break;
+                case "E":
+                    String at = taskArray[3];
+                    newTask = new Event(description, at, isDone);
+                    break;
+                default:
+                    throw new DukeException(String.format("Could not load task: %s", description));
+            }
+
+            taskList.add(newTask);
+        } catch (DukeException de){
+            prettyPrint(de.toString());
+        }
+
+    }
+
+    private static void storeToFile() {
+        try {
+            FileWriter fw = new FileWriter(STORAGE_FILE_PATH);
+            fw.write("");
+
+            for (Task t: taskList) {
+                fw.append(t.getStorageString());
+                if (!t.equals(taskList.get(taskList.size() - 1))) {
+                    fw.append("\n");
+                }
+            }
+
+            fw.close();
+        } catch (IOException e) {
+            prettyPrint("Failed to save all data!");
+        }
+
+    }
+
+    private static void createTask(String str, TaskType type){
+        try {
+            str = str.strip();
+            Task newTask;
+            String description = "";
+            String extraInfo = "";
+            switch(type) {
+                case TODO:
+                    if (str.strip().equals("")) {
+                        throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+                    }
+                    description = str;
+                    newTask = new ToDo(str);
+                    break;
+                case DEADLINE:
+                    if (str.strip().equals("")) {
+                        throw new DukeException("☹ OOPS!!! The description of a deadline cannot be empty.");
+                    }
+                    String[] strList = str.split(" /by ");
+                    description = strList[0];
+                    extraInfo = strList[1];
+                    newTask = new Deadline(description, extraInfo);
+                    break;
+                case EVENT:
+                    if (str.strip().equals("")) {
+                        throw new DukeException("☹ OOPS!!! The description of an event cannot be empty.");
+                    }
+                    strList = str.split(" /at ");
+                    description = strList[0];
+                    extraInfo = strList[1];
+                    newTask = new Event(description, extraInfo);
+                    break;
+                default:
+                    throw new DukeException("Type of task is not recognised!");
+            }
+            taskList.add(newTask);
+            String singulStr = "Now you have 1 task in the list";
```

Comment:
Perhaps using a standard word for the String variable name would make it easier to interpret its value and adhere more closely to the following requirement? 
> "Use correct spelling in names. Avoid 'texting-style' spelling."

I noticed the same issue in one or two other places too